### DISCUSSION TITLE
CORS 에러 발생

### DISCUSSION THREAD

**Discussion by hyoniiii** [WantedPreonboardingFE5team/pre-onboarding-7th-2-1-5#8] [General]
CORS에러 발생하신 분 없나요...?
API 관련 파일은
<img width="1377" alt="image" src="https://user-images.githubusercontent.com/97004730/199504798-a2c148f6-6871-4d19-933e-5beba88ccebb.png">
이렇게 구성되어있고

<img width="250" alt="image" src="https://user-images.githubusercontent.com/97004730/199618818-33eff02e-c251-4eb1-9631-61c2671fbefd.png">
Request URL이 Local로 들어가있어요😭

**Comment by hyeincha** [comment]
같은 사진이 두 장 들어간것 같은데 실제 요청 보내는 부분도 올려주시면 좋을 것 같습니다

**Reply by hyoniiii** [reply]
앗 그렇네요 같은 사진이 두 장 들어가있었네요!! 수정 했습니당

**Comment by hyoniiii** [comment]
아무래도 요청 쪽 문제인 것 같아서 apis.js 파일에서 getAll 부분에 full url을 넣어줬더니 요청이 정상작동했습니다!
그리고 나서 다시 위의 이미지와 같이 instance.js 파일에 baseURL을 넣어주었더니 잘 되더라고요...?
똑같은 코드 였는데 뭐가 문제였는지..는 잘 모르겠네요..😭

**Reply by jaeyeong815** [reply]
역시 알다가도 모를 코드세계.. ㅎ 어라 왜 되지? 어라 왜 안되지?의 연속 😤 암튼 고생하셨어욥
